src/Security/Member.php

```diff
@@ -57,7 +59,7 @@
  * @property string $DateFormat
  * @property string $TimeFormat
```

Comment:
Later in this file, Member::logOut() doesn't do a proper logout through the current IdentityStore; But tries to manually set cookies / session.